src/main/java/org/assertj/core/api/AbstractObjectArrayAssert.java

```diff
@@ -2572,7 +2572,7 @@ public SELF inBinary() {
     checkNotNull(filterOperator);
     Filters<? extends ELEMENT> filter = filter(actual).with(propertyOrFieldName);
     filterOperator.applyOn(filter);
-    return newListAssertInstance(newArrayList(filter.get())).as(info.description());
+    return newListAssertInstance(newArrayList(filter.get()));
```

Comment:
why not propagating the description ?

Reply:
It was brought by mistake in previous pull request and isn't covered by units.
also It prevents compilation on JDK9.
Do you think I should bring it back?